apps/examples/smp/smp_main.c

```diff
@@ -60,6 +60,12 @@
 #define CPU_ZERO(s) do { *(s) = 0; } while (0)
 #define CPU_SET(c,s) do { *(s) |= (1 << (c)); } while (0)
 
+#ifdef CONFIG_AMP
+enum {
+	CORE_0 = 0,
+	CORE_1, 
```

Comment:
Please check below comment and revise this..

Reply:
Done.